pkg/utils/binding/binding_test.go

```diff
@@ -315,3 +315,97 @@ func TestHasBindingFailed(t *testing.T) {
 		})
 	}
 }
+
```

Comment:
A nit (when I was resolving conflicts): the case on line 248 is essentially the same as the case on line 278.